### ISSUE TITLE
Error: On-device recognition is not supported for en-US

### ISSUE THREAD

**Issue by al-indigo** [sveinbjornt/hear#20]
I have Mac Mini (M1), MacOS Monterey 12.5.1 (21G83).
After attempt to launch `hear` I see the following:

```
./hear -s
ar-SA
ca-ES
cs-CZ
da-DK
de-AT
de-CH
de-DE
el-GR
en-AE
en-AU
en-CA
en-GB
en-ID
en-IE
en-IN
en-NZ
en-PH
en-SA
en-SG
en-US
en-ZA
es-419
es-CL
es-CO
es-ES
es-MX
es-US
fi-FI
fr-BE
fr-CA
fr-CH
fr-FR
he-IL
hi-IN
hi-IN-translit
hi-Latn
hr-HR
hu-HU
id-ID
it-CH
it-IT
ja-JP
ko-KR
ms-MY
nb-NO
nl-BE
nl-NL
pl-PL
pt-BR
pt-PT
ro-RO
ru-RU
sk-SK
sv-SE
th-TH
tr-TR
uk-UA
vi-VN
wuu-CN
yue-CN
zh-CN
zh-HK
zh-TW
```

But nevertheless I can not transcribe any audio with the following error:
```
./hear  -d -i ./test.m4a        
2023-03-09 19:20:01.767 hear[21203:2358806] Required assets are not available for Locale:en-US
Error: On-device recognition is not supported for en-US
```

Here are the settings of dictation (see the screenshot).

What am I missing?
![Screenshot 2023-03-09 at 19 20 28](https://user-images.githubusercontent.com/1831193/224070682-e81855ab-0faf-40e4-b50c-6418f4b3926d.png)

**Comment by pbmayu** [comment]
I had the same error. Once i wrapped the filenames in double quotes it worked.
e.g **hear -d -i "filename.mp3" > "output.txt"**

**Comment by jsonsmth** [comment]
Same here. Double quotes didn't work for me:

`hear -l en-US -i "./tmp/test.mp3"` works.

`hear -l en-US -d -i "./tmp/test.mp3"` does not:

`2023-04-09 08:56:37.217 hear[82015:1282124] Required assets are not available for Locale:en-US
Error: On-device recognition is not supported for locale 'en-US'`

**Comment by jsonsmth** [comment]
> 

I was able to resolve my issue by enabling dictation and waiting a few minutes:


![CleanShot 2023-04-09 at 09 08 18](https://user-images.githubusercontent.com/7754107/230774541-ae9c21b9-1f9d-4776-91b8-f96c1a75d40c.jpeg)

**Comment by sveinbjornt** [comment]
This is not a problem with "hear" per se, but with how macOS fetches locally needed resources for speech recognition.

**Comment by al-indigo** [comment]
1. Quotes: didn't help
2. Dictation is on (you can see it on the screenshot in the first message)

**Comment by sveinbjornt** [comment]
@al-indigo Can you use otherwise successfully use dictation in the OS?

**Comment by swoodford-clear** [comment]
I had this issue on M1 Macbook Pro, a reboot seemed to fix it

**Comment by al-indigo** [comment]
> @al-indigo Can you use otherwise successfully use dictation in the OS?

Yes, this Mac Mini M1 successfully works on dictation using both Logitech web-camera microphone and Sony WH-1000XM4. Maybe I can generate some diagnostics logs somehow?

**Comment by michael-2956** [comment]
You can fix it by first running hear without the "-d" flag on, so that MacOS fetches all the needed data, and then your usual command

**Comment by kepol** [comment]
Having the same issue on M1+Sonoma and tried:

1. Quotes
2. Enabling dictation
3. Running without -d and then with -d
4. Rebooting

but not of these worked for me.

**Comment by quinnburton** [comment]
I was able to get this to work in Ventura 13.5.1 by going to Finder > Edit > Start Dictation.

Once Dictation was running, I could then run the `hear -d -i filename.mp3 > output.txt` command in the terminal successfully

**Comment by fluxpiada** [comment]
it really works. you need to download the proper dictation language to your hd first (config > Translation Languages); and then add 
`-l en-AU` for Australian English and `hear` will choose that as dictation language for transcription. don't forget to add '-p' for punctuation!

**Comment by khanakia** [comment]
@fluxpiada how to download the languages ?

**Comment by smurp** [comment]
> it really works. you need to download the proper dictation language to your hd first (config > Translation Languages); and then add `-l en-AU` for Australian English and `hear` will choose that as dictation language for transcription. don't forget to add '-p' for punctuation!

I am in Canada and my Languages list (under the Dictation switch) had en-CA but not en-US in it at the start.

Thinking I just needed to add 'en-US' I just added it to to my languages list but only waited a couple of moments, perhaps it had not fully loaded.  This did not solve the problem.

I then added the `-l en-CA` switch to my command line and it worked fine.

**Comment by smurp** [comment]
> [@fluxpiada](https://github.com/fluxpiada) how to download the languages ?

![Image](https://github.com/user-attachments/assets/0402b415-5eeb-4a47-8616-cfb06cb1f40f)
